app/models/micropost.rb

```diff
@@ -0,0 +1,6 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+  default_scope -> { order(created_at: :desc) }
```

Comment:
Trả lời câu hỏi: default_scope có tác dụng như thế nào?
```suggestion
  default_scope -> {order created_at: :desc}
```

Reply:
scope thuộc vào các macro khác nên là thứ tự khai báo ở sau validates

Reply:
Dạ tác dụng là khi lấy dữ liệu db ra thì nó mặc định sắp xếp giảm dần